src/libraries/System.IO.Hashing/tests/XxHash128Tests.cs

```diff
@@ -0,0 +1,1086 @@
1Dtej07gSZA35GMesSOX6X2acTiUeS2LhGeWTGByPJwJMqe7u43noxXLZ4RFCQxv3OcOjvBrTQrt3HUITud2RefPVVVHpdXF2DN3kwBjFnonceJP6pk6MbkVv42oV0tgI9JHGWDOF5CU4mBYYZbKNv4QFgmvZYif7fAiFpuPKa8hW6jsiqVcWGRoUnJXrMYZp9zykNHGS76BcIzCgfSlF1jnZqeZcL2unmjoeyonwyic24B9n7YkSiWSEUmIgLXrqzGp82VXrcq9SRX9BAcn53U9jDQD8L9rksklY158asKiVJwGhj3EQuWpNqTipySKrr8R3xWLebpNCAnAlXckYTJqLZcrl2evBj3YmpfJWgOWkKMfGsluotOAnyLhr6psIe1PVNcYGUrilH1M6H9mF8LcWEuXvigL90roK4Kki3IDarCDSnc1mRzPf0I3v1ROuNNlgqbZNoMbL0TyMZDUq3wFzOf83emI8EHwhXagAd6LsfizcrSnVRraaOEYACDzqFBOjgtz1as0FKlaBSA67teTCKuoAp9XN4WBI0OsVPuY8NaiEkdvq3oPGahfqEHT988koGBqKJiv6GUeoTCQ3FsxOEXkVBlo46fimCNxIoqHKQKXoOoxfkJThKhGDrEK3R8iD4AyP5i3Whjakt0pmiLNTOoIrV0r7aqbvK7KA8NIZl8dXVjGQOtu1VyoLFfZ0nnWKMthJrroPqm83XcmulLWYxcG8lVGOLoilby0sxemMRDWveRqmrWx305h4n9y7wU5fYClZNlcsAC3xPPgDj5Mvf8bDEG4EIpZkvWGyUStN9vdkshPmm4vVMS9Fj25BTvPEix6ZL7MlRXYnQCOB1knysMTJsNPp7GN1YysFIFWRNvE4csYrJGkoeBKwMZVXvhLKgl2hqPyUfRJGGxBdTyuHItBHnQ2eXUtWuCqOXNktMOPReClFrP52DQlgQG3BDVHE0BZKc7qD7XTSAdpQmqkzWfT21xfHpwcWecFHDxacJPX97ULpeVAmiP0I3Z1dcI7KqjSbr1shDzyjPspnHgulEvDDzT1a81B3JHADw4jv6rlwn2lFfy3AN2T8nZu5R4hnJcd0PjxV8ylVYStP0aEqQmR5haflXffyH2DHc3w5BjFq59jdZ0aN8D4yapVuOJUPHJIJmFvIhXiZZmoWjGFnF3dKPgUz2AZlnbTINRNbSCSUq2J9q0ZKo3IwdtqyonEH02yrFTn2WGoL1nfepY3vxdXX6FMFKaeLiFs22rXfZ6RkIv5tEZCtJTisJNT055bKWdCcLLCn9jY0s2j3YJs8dHLepjhxe7uAFMytTVq8Y0SE65nhUV832leIZR9JrMmNsmdfI59ABGEh2f6CBJ3ZJFameh0co9BzYBtEtcsajgM6PCOElRJHO9d8xmBRIlmNhBhcGC4BH2oEtFo6tbRnC3FRPsTLsv2yiDNL9UJ3S6lBJfMhkbLdUXCgBjaEXz5TkbNQDaaujQWQkHAVMHIdQ0deOtlvMsYvBCszf9181qAoTDF2ahmDQAnTarUxEn9GN3MHumhjI8Eki3h0hLZzgtoo4ovZwCCZLBeeAX8ycud9oBnn2b6f17peUNURveHuFeAuFz7ZUtXez1BcoflaqyKoq6vh6MGVxnNthBArU4sBP352OLhXIFoEggIP2MGBGpj2LxsXSZhbHrHMRflHSAgrTDqkHKqfjRdSxdEHfXxMkdj4f0FVhZEgOf99IN00YXwXfet2gHQWHfRjsvjzkDTzLoZSy6HQPXYrKnIcgsoAtt4OGkO0LxKXoAZT8Lm9cF1eJHdsAWEMmssd73oV0WXA111iAhf2IOfQW65htyEvP2ljlN9BmA2PLXZGaq2EG0ozV6XqiZ1pgEyyQ2qxRdEoKv8VJQuoWSvK8j6at7V5RjChJVVwL96LdVgvxnSeinlRWWcf5jYljn7SvLPMV9F9fXGNhZGvvnQVfhU0cZtny8k7a1MO3gpnke0l1w03ZPgXVOMQH7DcIIy6IJSUNN5FqcubUATqdGWneXEi3HLYlcyO4CflRtGXO4vRSiAJSBla186vSiiJTe6L5t36t0G28hIIMkURDrAFjeB2VhLqVdqsmbDBxq4xxxrAi9KatciNCExmDk9JeHSrFeyjVkbw1SSb96Z4fSxtTyhpI7IiSP5vw9xqGBLUbpfewUaIfmXrwsAoLjurbE5W159x2bkL0b6XF9K0BMLsh7FJpl8Uh84XlDxYVUGYxL1jw9Cc29WAuy5uiCF6ThNw4p0NDlMslMMI6XJN4yz2bd7BJs4wlYFUwKqmwTVJAXoJIghgAX44ix9UxBcMjNWUqkIZa8Xaweh9pOrd7dhdWO2RMS25LD8yKSfan32deCGzcafjv6ZSy7MkiwYuviXpjlK4IJYd6wyitXts86MaP3ATXw472SHnnjLYJP9yBKuwBjdG5HfCUHGQwq5W6UQyoI7U5eszG62rJdr0YGkMuYbs0ayjWDf4gJxDz2UDwdkPH1fDOBIuQSiPcX8PRIELHfgwYseBhppC0PBDU6QAXjjxqRXz6iGHkNOOif67LS3tYOGif4jBPHEQLtbphuCCl2rXnKiRkxQb2LwEeSZVbPnzqlh5cN28bSLiFYpae03M7Cl7zC7oDz3XvthwgqfdZX224veXJzaEjsVOdXZKtNWaI8pf1J55Mq6FH5VRsHgpry86wXMVRrkty9UepTvrvkkHHprBAWuGQAanubxH2TRQrO8FukHKH0XfxxySbvN124hMBBNHysfWUyOhSoHl52AzkoARYTOkMS22MIGpLhuc8i7C2UVUraAZ5C3AhQITe2iVK6tEPHEy7tLFqHxrKpuZGKrc60pdkScSbRXsyF1kIWbc2JNDk4qv2QySbWPIIIz5aL9Pp8kT97QUiLFtdK8O2ghcHTFB8PqMKsg59Ex0Zlh2epGNCXqaqjkXjeMZxgLbqyrmfDMpNzOcgj4d66jhS0JwrFdoPAQOp1ODzJcLLP9eMhDl");
```

Comment:
Using similar feedback from XxHash3, can we manually wrap these strings? There's a bug in Visual Studio Code where strings this long will cause Visual Studio Code to completely lock up.

Some previous discussion at https://github.com/dotnet/runtime/pull/76641#issuecomment-1276336450.

Reply:
Ah, I was wondering why the original code was doing that, and of course, I wrote my generator without this 😅. No worries, I should be able to fix that.

Reply:
Should be fixed by commit 76d86cc